duckduckgo/duckduckgo-privacy-extension
# safari update atb

**Reviewer:**
@dharb 


## Description:

Update atb in the onNavigate event. 

## Steps to test this PR:

- Open the extension console network tab
- Try searches from the popup, omnibar, and DDG serp
- You should see an atb.js request in the network tab
- You shouldn't see atb.js requests when navigating to another domain

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
